### ISSUE TITLE
[enhancement] Settings and UI between Wear watch and AAPS 

### ISSUE THREAD

**Issue by Philoul** [MilosKozak/AndroidAPS#2214]
After using my Wear watch for a few days, I have several suggestions for improvement (grouped here):

**1) Treatment informations : When I use wizard from watch, I have no detailed informations in Treatment Tab of AAPS.**
=> I propose to add the equivalent of "Calc" link (could be named "Wear Wizard", may be "SMS" for SMS command if it's the same) with detailed information (carbs or correction entered and detailed calculation)
![Treatments informations](https://user-images.githubusercontent.com/52934600/69428912-a3737480-0d32-11ea-98f9-9c9b10cedc53.jpg)

**2) Correction Unit in calculation and wizard**
- In Calculation button of AAPS, the unit for Correction parameter is Insuline "U" (default value is empty = 0)
- In Wear Watch (if enable in settings), the unit is a percentage (default value is 100)
=> I propose to also use percentage in Calculation button (with of course 100 as default value)
We can that way use this parameter either as correction as currently, or to punctually deliver only a percentage of the bolus (If percentage parameter for bolus delivery in Preferences is at default value of 100)

**3) COB and IOB settings for calculation (AAPS) and wizard (Watch)**
- In Calculation Button (AAPS), it's impossible (for safety reasons) to enable COB (GA in image below) and disable IOB (IA in image below)
- In Wear settings for wizard, its possible as you can see below... (it's not my settings!!!)
=> I propose to have the same logic in Wizard settings of Wear watch (IOB is enable and unchangeable)
![Wear Wizard settings](https://user-images.githubusercontent.com/52934600/69429732-863fa580-0d34-11ea-967d-69113118de97.jpg)

**4) Watchface settings from AAPS**
- Most watchfaces display settings are made from the watch ... Except BGI which can only be set from AAPS (?)
=> I propose to move BGI setting (and may be the other "Display settings" of Wear if not to complicated) from AAPS to the watch settings.
![move to Wear settings](https://user-images.githubusercontent.com/52934600/69430132-52b14b00-0d35-11ea-823f-b7ee01e9f1cb.png)

**Comment by dlvoy** [comment]
Ad 4) it is because phone app formats and prepares output for watch, so we decide on contents at phone app (wear plugin) level. It allows wear implementation be way simpler and energy efficient. So moving it all to watch or phone will require some two-way sync - as xDrip does (partialy) - and what complicates a lot their wear version

**Comment by Philoul** [comment]
@dlvoy I don't understand your answer...
In my mind my 4) suggestion was only for "Display settings", not all wear settings...
I thought it could be possible (without a two-way-sync) to allways send to watch detailed IOB, detailed delta, BGI and predictions value, and then decide localy on watch to show or not these informations according to watch settings... (it works like that for other settings like Show IOB, Show COB, Show Delta or Chart Timeframe...)

**Comment by dlvoy** [comment]
Understood! It should be possible, only concern is to have it compatible between different versions of wear and main versions of AAPS.

**Comment by Philoul** [comment]
I have updated wiki page about settings in watch : https://androidaps.readthedocs.io/en/latest/EN/Configuration/Watchfaces.html#settings-in-wear-watch

Can you help me to update it with "Complications" ? (and also check if everything is ok on this page...)

I didn't understand the "Animation" setting... (I didn't see any change on my watch (Circle WF ?) when I switched it On or Off...)

**Comment by dlvoy** [comment]
> I have updated wiki page about settings in watch : https://androidaps.readthedocs.io/en/latest/EN/Configuration/Watchfaces.html#settings-in-wear-watch

Wow, that is impressive work! I will review it, work on _Complications_ section, and research what _Animation_ setting does.

**Comment by wootmasterslick** [comment]
I think you could better split this up into 4 separate issues
